roles/dnsmasq/defaults/main.yml

```diff
@@ -33,3 +30,46 @@ dnsmasq_min_replicas: 1
 
 # Custom name servers
 dnsmasq_upstream_dns_servers: []
+
+# Try each query with each server strictly in the order
+dnsmasq_enable_strict_order: false
+
+# Send queries to all servers
+dnsmasq_enable_all_servers: true
```

Comment:
`dnsmasq_enable_all_servers` should be set to false by default in order to fit the current behaviour